### PR TITLE
🌐 Add French definition for "monoculture"

### PR DESCRIPTION
**Pull request by CharlesMangwa**
Hi!

This PR aims to add the French definition for **"monoculture"**. 

A link has been added to the existing English definition (and vice versa).

Thanks in advance to the editor! ✨

### CODE REVIEW

File: 11ty/definitions/fr_FR/monoculture.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Monoculture
+slug: monoculture
+defined: true
+speech: nom
+---
+
+état d'une culture dans lequelle il y a un manque pertinent de diversité dans les objets culturels produits, tant par les organisations que par le grand public.
```
Comment (deammer) [inline 11ty/definitions/fr_FR/monoculture.md]: Minor typo: "état d'une culture dans l**a**quelle"

**Review by deammer — APPROVED** [review]
One small typo, but good to go otherwise 👏

**Comment by CharlesMangwa** [comment]
Thank you for proofreading, just fixed the typo!

File: 11ty/definitions/fr_FR/monoculture.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+title: Monoculture
```
Comment (tatianamac) [inline 11ty/definitions/fr_FR/monoculture.md]:
```suggestion
title: monoculture
```

**Review by tatianamac — APPROVED** [review]
🚀

**Pull request merged**